src/web/styles/buttons.scss

```diff
@@ -22,8 +22,8 @@ button {
 }
 
 .small-button {
-  padding: 0.5rem;
-  font-size: 1rem;
+  padding: 0.75rem;
+  font-size: 0.75rem;
```

Comment:
Kristen provided this CSS, but it looks too small to me and doesn't match Figma. Let's merge it as-is but then ask her to see if it looks right. Based on Figma the font-size should be 0.9375rem.

Reply:
Yes I agree, the text looks really small. I've changed it back to 1